README.md

```diff
@@ -9,6 +9,7 @@ This is an idiomatic Rust wrapper over [wgpu-native](https://github.com/gfx-rs/w
 
 ![Cube](etc/example-cube.png) ![Shadow](etc/example-shadow.png) ![MipMap](etc/example-mipmap.png)
 ![vange-rs](etc/vange-rs.png) ![Brawl](etc/brawl-attack.gif) ![GLX map](etc/glx-map.png)
+![Skybox](etc/skybox.gif)
```

Comment:
let's move it up to the other examples, and rename according to how others are named: `etc/example-skybox.gif`

Reply:
ok done